stack/update-global-hints.hs

```diff
@@ -80,29 +83,22 @@ scrapeGhcReleaseNotes vers = do
   pairs <- for vers myScrapeURL
   pure $ Map.fromList pairs
     where
-      url ver = T.unpack $ mconcat
-        [ "https://downloads.haskell.org/~ghc/"
-        , ver'
-        , "/docs/html/users_guide/"
-        , ver'
-        , "-notes.html"
-        ] where ver' = fromMaybe ver (T.stripPrefix "ghc-" ver)
+      url _ = "https://gitlab.haskell.org/bgamari/ghc-utils/-/raw/master/library-versions/pkg_versions.txt"
```

Comment:
In respect of this URL, looking at https://gitlab.haskell.org/bgamari/ghc-utils/-/tree/master/library-versions, I think things have moved on. When I put this URL into my browser, I get a 404 error.